content/en/docs/ambient/_index.md

```diff
@@ -1,7 +1,7 @@
 ---
 title: Ambient Mode
 description: Information for setting up and operating Istio in ambient mode.
-weight: 25
+weight: 17
```

Comment:
For Ambient Mode - Should we call it Ambient (Sidecar-less)?  Many folks don't know ambient is sidecar-less, why not make it clear for them.

Reply:
I'm going to take this PR without that change, and then we can figure out if we're going to want to use that terminology throughout. and how.  (It will be more than just here, and it causes the heading to be uncomfortably long)

<img width="311" alt="Screenshot 2024-04-30 at 2 31 46 PM" src="https://github.com/istio/istio.io/assets/132510/9672dc6e-2527-42d0-9d73-c30593535c20">